src/AddImageChecks.cpp

```diff
@@ -577,17 +591,22 @@ Stmt add_image_checks(Stmt s,
         }
     }
 
-    // Inject the code that checks the host pointers.
     if (!no_asserts) {
+        // Inject the code that checks the host pointers.
         for (size_t i = asserts_host_non_null.size(); i > 0; i--) {
```

Comment:
nit: is is worth adding an "append vector<Stmt> in reverse order" utility?

Reply:
Done